code/modules/clothing/head/mind_monkey_helmet.dm

```diff
@@ -0,0 +1,122 @@
+/obj/item/clothing/head/monkey_sentience_helmet
+	name = "Monkey mind-magnification helmet"
+	desc = "This helmet rapidly stimulates a monkey's mind to increase brain function, and in turn enables critical thinking skills."
+
+	flags_inv = HIDEHAIR
+	icon_state = "monkeymind"
+	base_icon_state = "monkeymind"
+	strip_delay = 100
+	clothing_flags = EFFECT_HAT | SNUG_FIT
+	COOLDOWN_DECLARE(message_cooldown) //It'll get annoying quick when someone tries to remove their own helmet 20 times a second
+	var/datum/mind/magnification = null ///A reference to the mind we govern
+
+/obj/item/clothing/head/monkey_sentience_helmet/update_icon_state()
+	. = ..()
+	icon_state = "[base_icon_state][magnification ? "_active" : ""]"
+
+/obj/item/clothing/head/monkey_sentience_helmet/equipped(mob/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_HEAD)
+		return
+	if(!ismonkey(user) || user.key)
+		to_chat(user, "<span class='boldnotice'>You feel a stabbing pain in the back of your head for a moment.</span>")
+		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+		if(isliving(user)) //I don't know what normally would force us to check this, but it's worth checking
+			var/mob/living/M = user
+			M.apply_damage(5,BRUTE,BODY_ZONE_HEAD,FALSE,FALSE,FALSE) //notably: no damage resist (it's in your helmet), no damage spread (it's in your helmet)
+			return
+		return
+	INVOKE_ASYNC(src, PROC_REF(poll), user)
+
+/obj/item/clothing/head/monkey_sentience_helmet/proc/poll(mob/living/carbon/monkey/user) //At this point, we can assume we're given a monkey, since this'll put them in the body anyways
+	if (user.stat) //Checks if the monkey is dead.
+		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE) //If so, buzz and do not poll ghosts
+		return
+	user.visible_message("<span class='warning'>[src] powers up!</span>")
+	playsound(src, 'sound/machines/ping.ogg', 30, TRUE)
+	var/list/candidates = pollCandidatesForMob(
+		Question = "Do you want to play as a mind magnified monkey?",
+		jobbanType = ROLE_MONKEY_HELMET,
+		role_preference_key = null,
+		poll_time = 100,
+		M = user,
+		ignore_category = POLL_IGNORE_MONKEY_HELMET)
```

Comment:
this pollignore is not necessary since the bantype is not used by anything else. poll ignores are autogenerated from the bantype or role preference.